copier/cli.py

```diff
@@ -71,7 +71,7 @@ def handle_exceptions(method, *args, **kwargs):
         return 1
     except UnsafeTemplateError as error:
         print(colors.red | "\n".join(error.args), file=sys.stderr)
-        return 2
+        return 4  # 0b100
```

Comment:
This might be a bit nitpicking, but IMHO this would be more helpful:

```suggestion
        # DOCS https://github.com/copier-org/copier/issues/1328#issuecomment-1723214165
        return 0b100
```

I'm based on the fact that somebody reading this code will want to understand why in the world is it written in bytes, and if they follow the link they'll maybe learn something interesting.

Reply:
Absolutely fine with me. :ok_hand:

Reply:
Done: f95585f